<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>567</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>567</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>567</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Построение графика</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(200, 200, 200);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="GraphWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>401</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>491</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0.034, y2:0, stop:0 rgba(255, 255, 111, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>y = k * x + b</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="work_btn">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0.034, y2:0, stop:0 rgba(255, 255, 111, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <property name="text">
        <string>Построить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Если этой части стало недостаточно, то нажмите на эту кнопку.</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>140</y>
      <width>111</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>a = </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>-99999</number>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_2">
       <property name="minimum">
        <number>-999999</number>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>b = </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox_3">
       <property name="minimum">
        <number>-9999</number>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>c = </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="dopBtn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0.034, y2:0, stop:0 rgba(255, 255, 111, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>Более сложные графики</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuThemes">
    <property name="title">
     <string>Themes</string>
    </property>
    <addaction name="actionBlack_theme"/>
    <addaction name="separator"/>
    <addaction name="actionStandart_theme"/>
   </widget>
   <addaction name="menuThemes"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBlack_theme">
   <property name="text">
    <string>Black theme</string>
   </property>
  </action>
  <action name="actionStandart_theme">
   <property name="text">
    <string>Standart theme</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
